tests/acceptance.py

```diff
@@ -82,7 +82,9 @@ def test_mypy(self):
             '(hint: "untyped_list = ...  # type: List[<type>]")'
         expected_py37 = '4:None|error: Need type annotation for ' + \
             '\'untyped_list\' (hint: "untyped_list: List[<type>] = ...")'
-        possible_results = expected, expected_py37
```

Comment:
Does it work if you only replace the `expected` value to be the same as `exp_expr`'s (without adding `exp_expr`)?

Reply:
It worked locally, Cadu. I updated accordingly 7632968f4aa3374dae8abaebeaeb91e5b5a11e51